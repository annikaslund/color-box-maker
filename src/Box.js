import React, { Component } from "react";
import "./Box.css";

class Box extends Component {
    render() {
        return <div style={ this.props.attributes }></div>
    }
}

export default Box;