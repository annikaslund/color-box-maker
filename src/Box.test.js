import React from "react";
import { mount } from "enzyme";
import toJson from "enzyme-to-json";
import Box from "./Box";

it("renders without crashing", function(){
    mount(<Box />);
});

it("matches snapshot", function(){
    let wrapper = mount(<Box/>);
    let serialized = toJson(wrapper);
    expect(serialized).toMatchSnapshot();
})