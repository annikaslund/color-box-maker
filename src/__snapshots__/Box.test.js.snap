// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches snapshot 1`] = `
<Box>
  <div
    style={
      Object {
        "backgroundColor": undefined,
        "height": "undefinedpx",
        "width": "undefinedpx",
      }
    }
  />
</Box>
`;
